lib/ravendb/documents/session/document_session.rb

```diff
@@ -9,19 +7,20 @@
 require "utilities/observable"
 
 module RavenDB
-  class DocumentSession
+  class DocumentSession < InMemoryDocumentSessionOperations
     include Observable
 
     attr_reader :number_of_requests_in_session
     attr_reader :documents_by_id
 
     def initialize(db_name, document_store, id, request_executor)
+      super()
```

Comment:
base class constructor takes some arguments in the java version
https://github.com/ravendb/ravendb-jvm-client/blob/48d6516935eb78009e000b6f72687318d7fb088f/src/main/java/net/ravendb/client/documents/session/DocumentSession.java#L52